docs/admin-guide.md

```diff
@@ -0,0 +1,42 @@
+# ChainPatrol
+
+### About this app
+
+The [ChainPatrol](https://chainpatrol.io/) mini-app gives your community members
+access to security tools to check URLs for scams and report malicious activity
+on your Discord server. Use the `/chainpatrol <sub-command>` commands to empower
+your community to protect themselves from phishing attacks and other scams.
+
+### Getting Started
+
+To get started, install the ChainPatrol mini-app from the
+[Collab.Land Marketplace](https://cc.collab.land/).
+
+Your community members can then start using `/chainpatrol check` to start
+checking URLs whenever they are unsure of a message that has been sent in your
+Discord server. If they find a scam, they can use the `/chainpatrol report`
+command to report the scam to ChainPatrol.
```

Comment:
does this mean that each community benefits from the reports made by other communities? i.e. global blocklist?

Reply:
Yup, it's a shared blocklist for all users, not only in Discord, but also Coinbase and Metamask use the data to protect their users as well.